### ISSUE TITLE
All Ingresses have the same secretName after jx upgrade ingress

### ISSUE THREAD

**Issue by wenzlaff** [jenkins-x/exposecontroller#182]
The command jx upgrade ingress recently seems to have some problems with handling TLS secrets. Currently all ingresses (jenkins, dockerregistry, …) are created with the same tls-secret (e.g. tls-docker-registry). 

jenkins:
```
  tls:
  - hosts:
    - jenkins.jx.example.com
    secretName: tls-docker-registry
```
docker-registry:
```
  tls:
  - hosts:
    - docker-registry.jx.example.com
    secretName: tls-docker-registry
```

**Comment by wenzlaff** [comment]
The fix seems to be available from v2.3.110 up, only, see:
https://github.com/jenkins-x/exposecontroller/blob/v2.3.110/exposestrategy/ingress.go

jx currently still using v2.3.109, seems we have to wait until https://github.com/jenkins-x/jenkins-x-platform/pull/5380 got merged.

**Comment by wenzlaff** [comment]
fixed with https://github.com/jenkins-x/exposecontroller/pull/184
